third-party/genuine-fantasy-press/forgotten-secrets/items.xml

```diff
@@ -0,0 +1,1324 @@
+		</setters>
+	</element>
+
+	<element name="Bloodstained Brush" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_DAGGER_BLOODSTAINED_BRUSH">
+		<description>
+			<p>With a blade crafted from hundreds of fine filaments, this dagger is reminiscent of a paintbrush. When you deal damage, you can impart a splash of that damage type’s color into the dagger, giving it a unique property that lasts until you finish a long rest. Each property only affects attacks made with this dagger. The dagger can contain multiple colors at the same time.</p>
+			<ul>
+			<li><b>Acid: Green</b> - Damage dealt can’t be healed until the target finishes a short or long rest.</li>
+			<li><b>Cold: Blue</b> - On a hit, the target’s movement speed is reduced by 5 feet for 1 minute. This does not stack.</li>
+			<li><b>Fire: Red</b> - The dagger’s base damage increases from 1d4 to 1d8.</li>
+			<li><b>Lightning: Yellow</b> - While wielding the dagger, your movement speeds increase by 10 feet.</li>
+			<li><b>Necrotic: Black</b> - Attacks made with advantage deal an extra 2d4 necrotic damage.</li>
+			<li><b>Psychic: Purple</b>  - While wielding the dagger, you gain immunity to the charmed and frightened conditions. </li>
+			<li><b>Radiant: White</b> - Damage dealt ignores resistance and immunity.</li>
+			</ul>
+		</description>
+		<setters>
+			<set name="keywords">dagger</set>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="dagger">Weapon</set>
+			<set name="rarity">Legendary</set>
+			<set name="weapon">Dagger</set>
+			<set name="attunement">true</set>
+		</setters>
+	</element>
+
+	<element name="Infinite Inkwell" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_INFINITE_INKWELL">
+		<description>
+			<p>This inkwell is strangely heavy and seems to pull at the light around it. You can use it to cast creation at will as a 9th level spell without expending a spell slot. You can’t cast the spell again until the duration of the previous cast has elapsed.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<set name="attunement">true</set>
+		</setters>
+	</element>
+
+	<element name="Ordinary Brush" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_ORDINARY_BRUSH">
+		<description>
+			<p>This looks like an ordinary paintbrush. You can use it to cast the cantrip fresh paint at will. </p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Common</set>
+		</setters>
+	</element>
```

Comment:
this would be a good description to add one of those `div reference` to the spell to put the spell description in there with the item description (there might be others but i noticed it here because i don't know the content from this source)